### ISSUE TITLE
It's possible to upvote a comment infinite times

### ISSUE THREAD

**Issue by 360fun** [Human-Connection/WebApp#330]
In relation to: https://github.com/Human-Connection/WebApp/issues/329
I just discovered that I can upvote a comment infinite times: this sounds like a bug. :P

**HOW TO REPRODUCE**

1. Click the arrow to add _+1 vote_
2. Refresh the page
3. Do it again...and again...and again :)

![image](https://user-images.githubusercontent.com/844090/47249965-b1154700-d41a-11e8-9380-48719e97ef18.png)

**Comment by roschaefer** [comment]
Yup, that's probably a bug. I was told, that not updating the comment vote count until a refresh is a **new** bug. I don't know what the previous behaviour was. I believe that there was never a comment vote limit or such thing and this is yet **another** bug.

**Comment by appinteractive** [comment]
The feature was never finished to be honest. Will be solved with a couple of other nasty things the next weeks

**Comment by 360fun** [comment]
Eheh, I had that sensation! :)

**Comment by Lulalaby** [comment]
Refresh of page is every time on vote needed to show vote. #Bug
